calico/reference/resources/felixconfig.mdx

```diff
@@ -64,6 +64,7 @@ spec:
 | ipv6Support                        | IPv6 support for Felix                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | true, false                                                                                             | boolean                                 | `true`                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| kubeNodePortRanges                 | This parameter holds a list of port ranges used for service node ports. Only used if felix detects kube-proxy running in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                  | A list of port ranges                   | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
It's not _a_ string, it's a _list of_ strings.

Reply:
Fixed